pipeline/data_merging/variant-merging.py

```diff
@@ -253,35 +281,38 @@ def add_leading_base(chr, pos, ref, alt, version="hg38"):
         brca_pos = pos - 1 - BRCA1[version]["start"]
     else:
         raise Exception("wrong chromosome number")
-    # correct error with when ref is empty string
-    if len(ref) == 0:
-        brca_pos += 1
-    else:
-        assert(seq[brca_pos:brca_pos + len(ref)] == ref)
```

Comment:
These lines created an off by one error for half of the variants. It would probably be a good idea to better understand what error they were intended to fix to make sure it hasn't returned since their removal.